packages/engine/src/scene/functions/loaders/TriggerVolumeFunctions.ts

```diff
@@ -24,6 +24,8 @@ export const deserializeTriggerVolume: ComponentDeserializeFunction = (
   entity: Entity,
   json: ComponentJson<TriggerVolumeComponentType>
 ): void => {
+  const xform = getComponent(entity, TransformComponent)
```

Comment:
we should keep the convention of camel case `<componentName>Component` we have elsewhere in the codebase for component instances